Core/KSP.cs

```diff
@@ -490,10 +490,14 @@ public override string ToString()
             return "KSP Install: " + gameDir;
         }
 
+        public bool Equals(KSP other)
+        {
+            return other != null ? gameDir.Equals(other.GameDir()) : base.Equals(other);
```

Comment:
This can be simplified, no?
If `other` is `null`, the last part could only possibly be true if `this` is also `null`,
and a `NullReferenceException` should've already been thrown by the runtime when trying to call `.Equals()` on it.

So you could also write
`return other != null && gameDir.Equals(other.GameDir())`

Reply:
OK, you convinced me. I wanted to be careful with this because the `base.Equals` call looked complicated, but it makes sense to simplify it.